automator/src/com/testsigma/automator/actions/web/generic/UploadFileAction.java

```diff
@@ -15,7 +15,7 @@ protected void execute() throws Exception {
     findElement();
     String downloadURL = getTestDataPropertiesEntity(TEST_STEP_DATA_MAP_KEY_TEST_DATA).getTestDataValuePreSignedURL();
     String fileName = String.format("%s_%s", System.currentTimeMillis(), FilenameUtils.getName(new URL(downloadURL).getPath()));
-    String filePath = String.format("%s%s%s", FileUtils.getTempDirectoryPath(), File.separator, fileName);
+    String filePath = String.format("%s%s%s", FileUtils.getTempDirectoryPath().endsWith(File.separator)?FileUtils.getTempDirectoryPath().substring(0,FileUtils.getTempDirectoryPath().length()-1):FileUtils.getTempDirectoryPath(), File.separator, fileName);
```

Comment:
@nixonakash01 You can store the value in FileUtils.getTempDirectoryPath() first and then use that value in this line in order to avoid repeated calls

Reply:
done bro, check latest commit